superdesk/io/__init__.py

```diff
@@ -12,21 +12,22 @@
 """Superdesk IO"""
 
 import logging
+
 import superdesk
-from superdesk.etree import etree
 from superdesk.celery_app import celery
-from superdesk.locators.locators import find_cities
-from superdesk.metadata.item import Priority
+from superdesk.errors import SuperdeskIngestError, AlreadyExistsError
+from .commands.add_provider import AddProvider  # NOQA
+from .ingest import IngestResource, IngestService
+
+registered_feed_parsers = {}
+allowed_feed_parsers = []
```

Comment:
this can use a view - `allowed_feed_parsers = register_feed_parsers.keys()` - it will be in sync